core-services/prow/02_config/_plugins.yaml

```diff
@@ -1586,6 +1597,17 @@ config_updater:
         build02:
         - ci
       name: cluster-profile-ovirt
+    cluster/test-deploy/kubevirt/*:
```

Comment:
I wonder if `kubevirt` here should be the name of the namespace in the CI cluster, i.e `kubevirt-ipi-provider` or whatever it was

Reply:
you right - I changed it to be kubevirt-ipi-provider